### ISSUE TITLE
SmartCity-Accessibility Server Side #420 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#265]
Build 'SmartCity-Accessibility Server Side' is failing!

Failed Tests :
```
No tests ran.
```

Changes since last successful build:
- [KoralChapnik] ef5e08bce0691f6871489fcf6ac69e7950bcaf9f - replaced user name with user in Review. put Location and Review manager

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side/420/)

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility Server Side' is failing!

Failed Tests :
```
No tests ran.
```

Changes since last successful build:
- [KoralChapnik] ef5e08bce0691f6871489fcf6ac69e7950bcaf9f - replaced user name with user in Review. put Location and Review manager

- [yaelAmitay] c58d31e6975f68ae9350c95f13d715456b4117f3 - make map clickable #264



[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side/421/)

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility Server Side' is failing!

Failed Tests :
```
No tests ran.
```

Changes since last successful build:
- [KoralChapnik] ef5e08bce0691f6871489fcf6ac69e7950bcaf9f - replaced user name with user in Review. put Location and Review manager

- [yaelAmitay] c58d31e6975f68ae9350c95f13d715456b4117f3 - make map clickable #264

- [Ariel Kolikant] 9429c3ca68db51ad1fafc5119f5c4f1bb61cf8ff - just added baisc working class #258



[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side/422/)

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility Server Side' is failing!

Failed Tests :
```
No tests ran.
```

Changes since last successful build:
- [KoralChapnik] ef5e08bce0691f6871489fcf6ac69e7950bcaf9f - replaced user name with user in Review. put Location and Review manager

- [yaelAmitay] c58d31e6975f68ae9350c95f13d715456b4117f3 - make map clickable #264

- [Ariel Kolikant] 9429c3ca68db51ad1fafc5119f5c4f1bb61cf8ff - just added baisc working class #258

- [Ariel Kolikant] 091def9844b02915bb024298dfa1303abf8cdb2a - search now understand the search query that it is given. need for it to

- [KoralChapnik] 1df2a5b0136d67dd48db7974e34f6916f9c00b37 - updating the helpfulness in uvoting / downvoting a user's comment #236



[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side/423/)

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility Server Side' is failing!

Failed Tests :
```
No tests ran.
```

Changes since last successful build:
- [KoralChapnik] ef5e08bce0691f6871489fcf6ac69e7950bcaf9f - replaced user name with user in Review. put Location and Review manager

- [yaelAmitay] c58d31e6975f68ae9350c95f13d715456b4117f3 - make map clickable #264

- [Ariel Kolikant] 9429c3ca68db51ad1fafc5119f5c4f1bb61cf8ff - just added baisc working class #258

- [Ariel Kolikant] 091def9844b02915bb024298dfa1303abf8cdb2a - search now understand the search query that it is given. need for it to

- [KoralChapnik] 1df2a5b0136d67dd48db7974e34f6916f9c00b37 - updating the helpfulness in uvoting / downvoting a user's comment #236

- [Ariel Kolikant] d2a1d38a0ed830f168c2649c1ed1ab8aa549e17b - service done, only simple improvements requires, maybe close #258

- [Ariel Kolikant] 2aca5a2683d747dbc76159f9922592d4dd7954f8 - add usage of map view in application close #269



[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side/424/)

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility Server Side' is failing!

Failed Tests :
```
No tests ran.
```

Changes since last successful build:
- [KoralChapnik] ef5e08bce0691f6871489fcf6ac69e7950bcaf9f - replaced user name with user in Review. put Location and Review manager

- [yaelAmitay] c58d31e6975f68ae9350c95f13d715456b4117f3 - make map clickable #264

- [Ariel Kolikant] 9429c3ca68db51ad1fafc5119f5c4f1bb61cf8ff - just added baisc working class #258

- [Ariel Kolikant] 091def9844b02915bb024298dfa1303abf8cdb2a - search now understand the search query that it is given. need for it to

- [KoralChapnik] 1df2a5b0136d67dd48db7974e34f6916f9c00b37 - updating the helpfulness in uvoting / downvoting a user's comment #236

- [Ariel Kolikant] d2a1d38a0ed830f168c2649c1ed1ab8aa549e17b - service done, only simple improvements requires, maybe close #258

- [Ariel Kolikant] 2aca5a2683d747dbc76159f9922592d4dd7954f8 - add usage of map view in application close #269

- [KoralChapnik] 0857fac0891367a4551258575f89ee573fa4c510 - implementing best reviews, need to add new tests for this class #228



[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side/425/)

**Comment by kapalex** [comment]
@koralchapnik 
the error seems to be, please have a look to see what might caused it in your push
```
05:27:41 Tests run: 4, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 10,024.004 sec <<< FAILURE! - in smartcity.accessibility.navigation.NavigationTest
05:27:41 displayMap(smartcity.accessibility.navigation.NavigationTest)  Time elapsed: 10,000.086 sec  <<< ERROR!
05:27:41 org.junit.runners.model.TestTimedOutException: test timed out after 10000 seconds
05:27:41 	at smartcity.accessibility.navigation.NavigationTest.displayMap(NavigationTest.java:79)
```

**Comment by koralchapnik** [comment]
@KaplanAlexander 
The problem is because I put some lines in LocationManager in comment because of the change from the user name to the user object in Review, and the Location Manager is used in the tests.
what to do ? put those tests in comment till you re-implement the locationManager ?

**Comment by kapalex** [comment]
I think put an ignore on the test and create an issue with you and me assigned to take care of it so we won't forget its there
also mention the commented out code there

**Comment by koralchapnik** [comment]
ok, no problem

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
